### ISSUE TITLE
Feature request: Open VSX

### ISSUE THREAD

**Issue by squili** [edgedb/edgedb-editor-plugin#22]
Open VSX is an open source version of the VS marketplace. It'd be nice to have this plugin published there as well, since forks like VSCodium aren't allowed to use the official marketplace. https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by 1st1** [comment]
@ambv can you take a look into this?

**Comment by filiptronicek** [comment]
Hi there 👋! I'm Filip and work on the Open VSX project.
Wanted to chime in here to provide any help / assistance you folks may need with publishing the extension to Open VSX. In our team, we have made the [Publishing an extension to Open VSX on your own doc](https://github.com/open-vsx/publish-extensions/blob/master/docs/direct_publish_setup.md) and I think it might help here 🙂. 

Feel free to ping me if you need anything!

cc @1st1 @ambv

**Comment by mcgrealife** [comment]
+1
I am using edgedb in a gitpod container, and the extension is not available unless published to Open VSX.

Thanks @filiptronicek! cc @1st1 @ambv

**Comment by Lancelotbronner** [comment]
I tend to prefer VSCodium to VSCode because of how much snooping around VSCode does, was this feature still considered?

**Comment by 0c12a4c9-4c00-4211-8531-353dbff93969** [comment]
Yes, please. This would help me to install the extension in VSCodium

**Comment by chrisemke** [comment]
is there any news about this?

**Comment by JoshZA** [comment]
👍 This would allow us to use VSCodium

**Comment by aljazerzen** [comment]
Closed by 29acde479b86c48f100df34af8d87e16a0ec3b7a

Extension is published at https://open-vsx.org/extension/gel/edgedb
